kura/org.eclipse.kura.wire.ai.component.provider/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,20 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Kura Wires AI Component provider
+Bundle-SymbolicName: org.eclipse.kura.wire.ai.component.provider
+Bundle-Version: 1.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-Vendor: Eclipse Kura
+Bundle-License: Eclipse Public License v2.0
+Bundle-Category: Kura Wires
+Import-Package: org.eclipse.kura;version="1.6.0",
+ org.eclipse.kura.ai.inference;version="[1.0,2.0)",
+ org.eclipse.kura.configuration;version="[1.1,2.0)",
+ org.eclipse.kura.type;version="[1.0,2.0)",
+ org.eclipse.kura.wire;version="[2.0,3.0)",
```

Comment:
If your bundle implements an interface marked as `ProvideType` then the package that provides that interface should be inported with a range like `[current_major.current_minor, current_major.current_minor+1)`